Chapter2/TodoListAPI/Startup.cs

```diff
@@ -30,6 +30,20 @@ public void ConfigureServices(IServiceCollection services)
             services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
                 .AddProtectedWebApi(Configuration);
 
+            // Uncomment this section if you would like to validate ID tokens for allowed tenantIds
+            // services.Configure<JwtBearerOptions>(JwtBearerDefaults.AuthenticationScheme, options =>
```

Comment:
Great work 👍 
I could suggest that we may consider the MIW approach to extend token validation.

Reply:
Thanks Shama! Can you share a link for me to see that approach?

Reply:
You can find the[ implementation here](https://github.com/Azure-Samples/active-directory-dotnet-native-aspnetcore-v2/blob/ea2e7782bf8b6c7ddd1e52fd80ee3c6737de128d/3.-Web-api-call-Microsoft-graph-for-personal-accounts/TodoListService/Startup.cs#L29)  or there is a [stackoverflow link](https://stackoverflow.com/questions/62161703/azure-sso-ontokenvalidated-is-not-working/62210028#62210028).

Reply:
got it!